R/datamapping-boxwhisker.R

```diff
@@ -0,0 +1,67 @@
+#' @title BoxWhiskerDataMapping
+#' @docType class
+#' @description  Class for Box Whisker Mapping
+#' @export
+BoxWhiskerDataMapping <- R6::R6Class(
+  "BoxWhiskerDataMapping",
+  inherit = XYDataMapping,
+  public = list(
+    quantiles = NULL,
+    outliers = NULL,
+    yMin = NULL,
+    lower = NULL,
+    middle = NULL,
+    upper = NULL,
+    yMax = NULL,
+
+    initialize = function(...,

```

Comment:
@pchelle usually, ... are the last parameters, and not the first. Can we adhere to the convention?

Reply:
Yeah I understand
How does that look in intellisense?
for this type of use case,s I would destructure x, and y and add them explicitely before if this is not the case already